### DISCUSSION TITLE
My custom colours don't work after latest update

### DISCUSSION THREAD

**Discussion by cg00001** [umbrellaplug/umbrellaplug.github.io#188] [General]
Should I uninstall Umbrella, deleting settings.xml too and install again?
![070717](https://user-images.githubusercontent.com/38960370/211982612-e6fef664-150c-4e39-bdfc-328d29b5508e.jpg)
![070833](https://user-images.githubusercontent.com/38960370/211982616-71f85d2c-8a99-4293-b1f1-42e89a5f1992.jpg)

**Comment by cg00001** [comment]
I uninstalled Umbrella, installed repo from scratch and I checked Umbrella repo, and it only has CocoScrapers in Program addons inside...
I'm using Nexus, maybe the repo upgrade 3 hours ago broke it?

**Reply by umbrellaplug** [reply]
If you’re on kodi 20 the latest update switched the colors for the new color button setting. You likely need to a select your colors again. If you’re on a skin and cannot see the colors at all. Switch to estuary set your colors then switch back. 

I put a request in for jurial asking if he could add support for colorbutton as well. This is a new core kodi settings component.

**Reply by cg00001** [reply]
OK, will do, what about your repo -yes Kodi 20 RC2 Nexus?
This is all I can now see inside it (Program addons):

![153650](https://user-images.githubusercontent.com/38960370/212081244-83ac5030-0010-4cf3-866d-f5351b44bcb1.jpg)

**Reply by umbrellaplug** [reply]
I'm going to roll the update back. And force the repo back to only one version. Should be fixed in the next little bit.

**Reply by cg00001** [reply]
Using Estuary showed the color lines like you mentioned so I could change them, thanks.
![154328](https://user-images.githubusercontent.com/38960370/212082830-525248ea-4d99-4095-a4d4-cceb8f59a2db.jpg)

**Comment by cg00001** [comment]
OK, I downloaded the addon from site, no color customizing on settings any more, so that explains things.
![085908](https://user-images.githubusercontent.com/38960370/211999092-457200e8-0f1c-4c98-a9ee-8094fda0aa79.jpg)

**Comment by cg00001** [comment]
I just updated the repo, addon is there,  and color lines in settings are showing again on my skin, thank you.

![165757](https://user-images.githubusercontent.com/38960370/212100763-36b36635-2e10-4337-83b2-6770c3e754a0.jpg)

**Comment by cg00001** [comment]
I changed colors using latest repo/addon after uninstalling previous, and changes don't apply, default colors remain. Tried Estuary too.

![173940](https://user-images.githubusercontent.com/38960370/212112173-bce01046-ad87-4865-9ff1-48daac11d7f8.jpg)

**Reply by cg00001** [reply]
Latest update fixed custom colors for good. Thanks again.
![220308](https://user-images.githubusercontent.com/38960370/212169494-8229e0f7-048a-414b-83af-6dbe8588a072.jpg)
